.github/system_tests/test_daemon.py

```diff
@@ -200,15 +200,15 @@ def validate_cached(cached_calcs):
             print_report(calc.pk)
             valid = False
 
-        if '_aiida_cached_from' not in calc.extras or calc.get_hash() != calc.get_extra('_aiida_hash'):
+        if '_aiida_cached_from' not in calc.base.extras or calc.get_hash() != calc.base.extras.get('_aiida_hash'):
             print(f'Cached calculation<{calc.pk}> has invalid hash')
             print_report(calc.pk)
             valid = False
 
         if isinstance(calc, CalcJobNode):
-            original_calc = load_node(calc.get_extra('_aiida_cached_from'))
-            files_original = original_calc.list_object_names()
-            files_cached = calc.list_object_names()
+            original_calc = load_node(calc.base.extras.get('_aiida_cached_from'))
+            files_original = original_calc.base.repository.list_object_names()
+            files_cached = calc.base.repository.list_object_names()
```

Comment:
This was apparently missed, but I don't think I saw the deprecation warning for this. Are those warnings working properly?

Reply:
No, the warnings are probably not "activated" for these separate system tests. Obviously, just need to set the global variable (separate PR?)

Reply:
Uhm, it seems like `test_daemon.py` is not being run at all anymore :sweat_smile: It accidentally got removed in 5d2f15380417ce963544e209346b31eb37b0490d when the RPN workchains were moved to the nightly build. I will open a PR to reinstate them